src/TextEditor/editor.cpp

```diff
@@ -1,5 +1,7 @@
 #include "editor.h"
 
+#include <QScrollBar>
```

Comment:
Accidental include? Don't think these changes require this.

Reply:
I was experimenting with scrolling and forgot to remove it. fixed